src/EditorFeatures/Core/InheritanceMargin/AbstractInheritanceMarginService.cs

```diff
@@ -87,13 +87,16 @@ internal abstract class AbstractInheritanceMarginService : IInheritanceMarginSer
 
         private static bool CanHaveInheritanceTarget(ISymbol symbol)
         {
-            if (symbol.IsStatic)
+            if (symbol is INamedTypeSymbol)
             {
-                return false;
+                return !symbol.IsStatic;
```

Comment:
❔ Can static types implement interfaces with only static abstract members?

Reply:
https://github.com/dotnet/csharplang/blob/main/proposals/static-abstracts-in-interfaces.md
At least from the existing document, I don't think static type could do that.